samples/nrf9160/aws_fota/prj.conf

```diff
@@ -51,3 +51,6 @@ CONFIG_FLASH=y
 
 # Enable Log
 CONFIG_LOG=y
+CONFIG_LOG_STRDUP_MAX_STRING=1500
+CONFIG_LOG_STRDUP_BUF_COUNT=10
```

Comment:
Guys, this will result in extra 15K of RAM only for the logger buffers, do we really want that? How big is the download speed impact in non-deffered mode?

If we stick to the buffer increase though, I'd check with stack sentinel if we don't exceed the logger stack; I've noticed that this buffer is allocated on a stack in one function, I'm not sure when/if it is called though.

Reply:
As said I'm fine with either. It's just to showcase in-case someone decides to turn on max debug log output from AWS FOTA. 

I'm fine with putting a more sensible MAX_STRING size. What's not so easy to see is what `~` means that the string is  cut-off. The `log_strdup alloc` failed is an error which is well defined and easily searchable in the documentation. I could try to document it in a sensible way.

I would say the speed impact in non-deferred mode is negligible especially when using a low bandwith mode like NB-IoT

Reply:
Reduced the max string size.

Reply:
Is this size acceptable @rlubos ? or do you suggest going back to immediate logging?